apps/etterlatte-vedtaksvurdering-kafka/src/main/kotlin/no/nav/etterlatte/vedtaksvurdering/rivers/AldersovergangRiver.kt

```diff
@@ -56,14 +58,25 @@ class AldersovergangRiver(
             ),
         ) {
             if (step == VedtakAldersovergangStepEvents.IDENTIFISERT_SAK.name) {
+                val hendelseData = mutableMapOf<String, Any>()
+
                 val behandlingsdato = packet.dato
                 logger.info("Sjekker løpende ytelse for sak $sakId, behandlingsmåned=$behandlingsdato, dryrun=$dryrun")
 
                 val loependeYtelse = vedtakService.harLoependeYtelserFra(sakId, behandlingsdato)
                 logger.info("Sak $sakId, behandlingsmåned=$behandlingsdato har løpende ytelse: ${loependeYtelse.erLoepende}")
+                hendelseData["loependeYtelse"] = loependeYtelse.erLoepende
+
+                if (loependeYtelse.erLoepende) {
+                    val januar2024 = LocalDate.of(2024, Month.JANUARY, 1)
```

Comment:
Mistenkjer at vi bør sende med aktuell månad i pakka heilt frå starten av. Ser ut som det her er litt WIP, og det er for så vidt greitt nok, men til slutt

Reply:
Aktuell behandlingsmåned for hendelsen sendes med fra starten av, men akkurat denne sjekken gjelder jo kun 1.1.2024.

Reply:
Oppdaterer variabelnavnet til noe mer meningsfylt, á reformtidspunktBarnepensjon2024